src/BloomBrowserUI/bookEdit/readerSetup/readerSetup.ui.ts

```diff
@@ -758,7 +758,12 @@ function attachEventHandlers(): void {
       return false;
     });
 
-    $('#allowed-words-file-div').onOnce('mouseenter', function() { $(this).find('a').show(); });
+    $('#allowed-words-file-div').onOnce('mouseenter', function() {
+      var title = localizationManager.getText('ReaderSetup.RemoveWordList', 'Remove from this stage');
```

Comment:
Good point! The whole idea is that localizationManager has already done its work! No point calling it again. I'll fix it.